api/v1alpha1/observatorium_types.go

```diff
@@ -260,6 +267,12 @@ type RuleSpec struct {
 	Version string `json:"version,omitempty"`
 	// VolumeClaimTemplate
 	VolumeClaimTemplate VolumeClaimTemplate `json:"volumeClaimTemplate"`
+	// RulesConfig configures rules from the configmaps
+	// +optional
+	RulesConfig []RuleConfig `json:"rulesConfig,omitempty"`
+	// AlertmanagersURL
+	// +optional
+	AlertmanagersURL []string `json:"alertmanagersURL,omitempty"`
```

Comment:
Comparing to the Prometheus Operator: https://github.com/prometheus-operator/prometheus-operator/blob/f6aba9769f1a9c6c199731cc63d21f5f0c909d25/pkg/apis/monitoring/v1/types.go#L394-L397

However, in this case we're talking about the Alertmanagers running somewhere outside the cluster that this instance of the Operator & Observatorium are running. Is that assumption correct?

Reply:
In our case, the alertmanager is running in-cluster that this instance of the Operator & Observatorium are running